data/phi0119/phi006/__cts__.xml

```diff
@@ -1,11 +1,11 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0119.phi006" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0119">
             <ti:title xml:lang="lat">Casina</ti:title>
             <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0119.phi006.perseus-eng2" workUrn="urn:cts:latinLit:phi0119.phi006">
-              <ti:label xml:lang="eng">Casina, or The Stratagem Defeated</ti:label>
-              <ti:description xml:lang="eng">Plautus, Titus Maccius, creator; Riley, H. T. (Henry Thomas), 1816-1878, translator</ti:description>
+              <ti:label xml:lang="mul">Casina, or The Stratagem Defeated</ti:label>
+              <ti:description xml:lang="eng">Plautus Riley, H. T. , translator</ti:description>
```

Comment:
unexpected space

Reply:
hey @lcerrato. Yeah that's just because I had started to change out the ti:descriptions and was eliminating extraenous info using find and replace across the files. I was going to do a second pull request with all the bibliographic info in the cts_.xml files changed on Monday and so forth with the other changes in Plautus. But I can fix it here before you approve if you like!